rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -393,21 +393,27 @@
         }
       }
     },
-    "library": {
-      "title": "Library",
-      "description": "Library affiliation.",
-      "additionalProperties": false,
-      "type": "object",
-      "required": [
-        "$ref"
-      ],
-      "properties": {
-        "$ref": {
-          "type": "string",
-          "pattern": "^https://ils.rero.ch/api/libraries/.*?$",
-          "form": {
-            "remoteOptions": {
-              "type": "libraries"
+    "libraries": {
+      "title": "libraries",
+      "description": "Library affiliations for the librarian.",
+      "type": "array",
+      "uniqueItems": true,
+      "minItems": 1,
+      "items": {
+        "title": "Library",
+        "type": "object",
+        "required": [
+          "$ref"
+        ],
+        "properties": {
+          "$ref": {
+            "title": "Library URI",
+            "type": "string",
+            "pattern": "^https://ils.rero.ch/api/libraries/.*?$",
+            "form": {
+              "remoteOptions": {
+                "type": "libraries"
+              }
```

Comment:
maybe add
```
"templateOptions": {
  "label": ""
}
``` 
to have a better editor display (removing the label "library URI")

Reply:
done